packages/storage/src/append_store.rs

```diff
@@ -60,6 +79,40 @@ impl<'a, T: Serialize + DeserializeOwned, Ser: Serde> AppendStore<'a, T, Ser> {
 }
 
 impl<'a, T: Serialize + DeserializeOwned, Ser: Serde> AppendStore<'a, T, Ser> {
+    fn as_slice(&self) -> &[u8] {
+        if let Some(prefix) = &self.prefix {
+            prefix
+        } else {
+            self.namespace
+        }
+    }
+
+    fn _page_from_position(&self, position: u32) -> u32 {
```

Comment:
Seems like the current convention is to not prefix function names with a `_`, just it not being `public` is enough